clusters/resource_cluster.go

```diff
@@ -498,5 +498,4 @@ func resourceClusterDelete(ctx context.Context, d *schema.ResourceData, c *commo
 
 func init() {
 	common.RegisterResourceProvider(compute.ClusterSpec{}, ClusterSpec{})
-	common.RegisterResourceProvider(compute.Library{}, LibraryResource{})
```

Comment:
Removing it from the registry because the only customization in `library` is the added required `cluster_id` field, it is only needed in `library` and not anywhere else.

Reply:
Add this back in after making cluster ID field resource-specific.